src/index.html

```diff
@@ -16,17 +29,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <div class="search-panel">
+      <form
+        class="search-form big"
+        data-qa="keypress"
+      >
+        <input
+          class="search__input search__input--big"
```

Comment:
The class name `search__input--big` follows BEM naming conventions, which is correct. Ensure consistency across all class names.